scripts/test.sh

```diff
@@ -61,12 +61,14 @@ then
   echo "     bash $0"
   echo ""
   return 1
-fi
+fi 
```

Comment:
Not sure why this line was changed? It shouldn't have the extra space at the end.